src/PDFA-3-0/Cert/_reference/resources/medmij-pdfa-DocumentReference-kwalificatie5.xml

```diff
@@ -48,6 +48,7 @@
       <reference value="#p1"/>
       <display value="A.F. Snijder"/>
    </author>
+   <description value="Example PDF not in Bonary"/>
```

Comment:
Typo: Binary.
Moreover, for this resource the `.description` and `.content.attachment.title` do not align. Why is `.description`not equal to 'Discharge Summary'?

Reply:
Done